app/models/search.rb

```diff
@@ -0,0 +1,7 @@
+class Search
+  include ActiveModel::Model
+
+  attr_accessor :date_of_birth, :last_name
+
+  validates :date_of_birth, :last_name, presence: true
```

Comment:
This hasn't quite done the trick — 

- An instance of search isn't being passed to the view, so no validation errors appear
- If any of the date fields are blank, you get a parse error. We'll need to validate the presence of these fields prior to parsing a date out of them.